src/StreamrClient.js

```diff
@@ -372,11 +310,29 @@ export default class StreamrClient extends EventEmitter {
         return this.connection.enableAutoDisconnect(...args)
     }
 
+    getAddress() {
```

Comment:
maybe instead of these functions, everyone who wants those things could just call `client.ethereum.getSigner()` etc.? Not all of that is really relevant to most user needs, so no need to wrap more than just `getAddress()` maybe

Reply:
Good thought! Let's do that.